package.json

```diff
@@ -135,6 +135,7 @@
     "koa-socket-2": "^1.0.17",
     "lru-cache": "^4.1.1",
     "mousetrap": "^1.6.1",
+    "node-persist": "^3.0.4",
```

Comment:
DB dependencies are not automatically added, but the user is responsible for adding it themselves.

Can you update docs/storage.md?

Reply:
Thanks!  I moved node-persist to devDependencies (I think tests/CI will fail otherwise), and I updated storage.md.